core/src/main/java/io/grpc/MethodDescriptor.java

```diff
@@ -267,7 +267,8 @@ public RespT parseResponse(InputStream input) {
   }
 
   /**
-   * Convert a request message to an {@link InputStream}.
+   * Convert a request message to an {@link InputStream}. gRPC will close the InputStream
```

Comment:
This seems over-specifying, as that isn't true if someone else calls the method. You could say "The returned InputStream should be closed by the caller."

Reply:
fixed